ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -27,13 +27,6 @@
       }
     },
 
-    data: () => ({
```

Comment:
Why are we deleting these from here?

Reply:
Loading and sessions should be handled by Vuex. For example the loading data gets replaced by `currentInteractiveDataLoading` vuex getter. And the other data properties are also replaced and driven by Vuex.